analytics/terraform/spark-k8s-operator/addons.tf

```diff
@@ -417,7 +417,7 @@ module "eks_blueprints_addons" {
     }
   }
   karpenter = {
-    chart_version       = "v0.34.0"
+    chart_version       = "1.0.0"
```

Comment:
please use the latest version 1.0.6

Reply:
I think I was able to merge all the upstream changes and keep the versions you mention here (Addons 1.34.0 and karpenter 1.0.6).

Reply:
awesome, this LGTM!